src/main/java/org/jukeboxmc/event/player/PlayerBucketEvent.java

```diff
@@ -76,7 +76,7 @@ public Block getPlacedBlocK() {
      * @param placedBlocK which will be placed
      */
     public void setPlacedBlocK(Block placedBlocK) {
```

Comment:
Hier bitte das K klein machen

Reply:
yeah, sorry due to variable name change xD